### PR TITLE
fix(test): lint lll fix

### PR DESCRIPTION
**Pull request by skrdgraph**
## Problem
 `golangci-lint` tests are failing. The `lll` linter which addresses our `line-length` is reporting issues on a few files. 

## Solution
- fix up lll issues using `golines -m 120 -w <GO_FILE>`
- `golangci-lint run | grep lll` <- this will show where the tests are failing

#### Results 

`lint` error count reduces by ~50+ with this change. This only fixes `lll` issues

- ##### before
  ```
   > golangci-lint run | wc -l
  WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive. 
       291
  ```
- ##### after
  ```
   > golangci-lint run | wc -l
  WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive. 
       215
  ```

### CODE REVIEW

**Comment by skrdgraph** [comment]
@matthewmcneely & @MichelDiz - it's cleaned up and free of any `git` rebase issues. I had to do this on a new branch.

File: worker/backup_ee.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build !oss
 // +build !oss
```
Comment (skrdgraph) [inline worker/backup_ee.go:2]: @matthewmcneely - this change seems to have happened from `auto-fix` code, should I undo this?

**Review by meghalims — APPROVED** [review]
Looks good 👍

**Review by meghalims — APPROVED** [review]
Changes look good 👍

**Review by meghalims — COMMENTED** [review]
The changes to the go modules look good. What does the check failures  entail?

File: graphql/resolve/query_rewriter.go [hunk 2]
```diff
@@ -1211,7 +1212,9 @@ func buildAggregateFields(
 				}
 				aggregateChild := &gql.GraphQuery{
 					Alias: aggregateField.DgraphAlias() + "_" + f.DgraphAlias(),
-					Attr:  strings.ToLower(function) + "(val(" + "" + f.DgraphAlias() + "_" + constructedForField + "Var))",
+					Attr: strings.ToLower(
+						function,
```
Comment (matthewmcneely) [inline graphql/resolve/query_rewriter.go:1216]: This still bothers me :).... how about

```golang
				aggregateChild := &gql.GraphQuery{
					Alias: aggregateField.DgraphAlias() + "_" + f.DgraphAlias(),
					Attr: strings.ToLower(function) + 
                                             "(val(" + "" + f.DgraphAlias() + "_" + constructedForField + "Var))",
				}

```

**Review by matthewmcneely — COMMENTED** [review]
All good except for that whack reformat in query_rewriter.go

File: graphql/resolve/query_rewriter.go (hunk 2 above)
Comment (skrdgraph) [reply graphql/resolve/query_rewriter.go:1216]: I will fix it up tomorrow, current plan is to get this to a good shape as soon as we have a minimal test-harness running (which we have atm ~40% coverage, and if we open up a few more folders - we will be in a better shape to accept changes more confidently ... particularly for `GraphQL` folder.

File: worker/backup_ee.go (hunk 1 above)
Comment (skrdgraph) [reply worker/backup_ee.go]: https://github.com/dgraph-io/dgraph/pull/8191/commits/1161c79cddcdcd5b305085841d56f2f5c0f2f3b7 addressing this here

File: graphql/resolve/query_rewriter.go (hunk 2 above)
Comment (skrdgraph) [reply graphql/resolve/query_rewriter.go]: @matthewmcneely thanks for providing the new structure .. I have manually fixed it here https://github.com/dgraph-io/dgraph/pull/8191/commits/e378950dc958730782f23a61f1dbe19a3a3c7a8a

**Comment by skrdgraph** [comment]
I have addressed all the comments, waiting for tests to pass before merging.

**Comment by skrdgraph** [comment]
@matthewmcneely after a bunch of issues, I've addressed your points & I've also fixed some test failures which came up because of `lll` fixes. 

Im merging this one ... 

With this we will have most basic tests passing!

**Pull request merged**